pkg/config/config.go

```diff
@@ -181,6 +181,9 @@ type Metrics struct {
 	// Defines the metrics port that the app should be configured to listen on for
 	// metric traffic.
 	Port int `mapstructure:"port"`
+
+	// How often (in seconds) to run queries to collect some metrics
+	CollectionFrequency int `mapstructure:"collection_frequency"`
```

Comment:
this should also be in config.yaml.example

Reply:
yes!